generators/new-wp/templates/.gitignore

```diff
@@ -0,0 +1,31 @@
+### Ignore all Files
```

Comment:
Also all the templates should be placed without, why? since this is the standard used in most of the yeoman generator and to avoid conflicts with other files as well. Take a look at the php-lib example https://github.com/moxie-leean/generators/tree/master/generators/php-lib/templates